README.md

```diff
@@ -6,7 +6,7 @@
 
 ## TL;DR?
 
-* [Start Developing Your First Helix Project in 60 Seconds](https://www.project-helix.io/)
+* [Start Developing Your First Helix Project](https://main--acom-kit--auniverseaway.hlx.page/tutorial)
```

Comment:
Note this tutorial needs to be updated to Helix 3.

Reply:
cc @auniverseaway